lib/generator.js

```diff
@@ -13,6 +13,8 @@ const Ajv = require('ajv');
 const filenamify = require('filenamify');
 const git = require('simple-git/promise');
 const npmi = require('npmi');
+const semver = require('semver');
+const packageJson = require('../package.json');
```

Comment:
I think better would be to read the version from environment variable `process.env.npm_package_version` as we do it here already https://github.com/asyncapi/generator/blob/master/package.json#L18

Reply:
Why is this approach better than reading from the package.json file? It's not obvious that this environment variable exists. Also, if someone uses Yarn (or another package manager), does this environment variable exist too?

Reply:
This is officially supported variable https://docs.npmjs.com/misc/scripts#packagejson-vars

At the end is it alwasy better to not read file if not needed.

Yarn also supports it https://classic.yarnpkg.com/en/docs/cli/version/#toc-version-lifecycle

Reply:
I don't think this environment variable is available outside `preversion`, `version`, and `postversion`.

![Screen Shot 2020-04-28 at 11 15 02](https://user-images.githubusercontent.com/242119/80469997-a3128180-8941-11ea-952f-ac5f0da99c00.png)

Reply:
Definitely they're not available. Check docs here: https://docs.npmjs.com/misc/scripts#environment. It's only when running a package script.

Reply:
ok, forget it, it is too risky, works only if you run code with `npm` script, it will only complicate the development flow